containres/ideasdk/server/soca_server.py

```diff
@@ -649,7 +649,7 @@ def initialize(self):
             )
             self.apply_sanic_extensions(self._unix_app)
 
-        if self.options.enable_metrics:
+        if False:  # todo self.options.enable_metrics:
```

Comment:
should probably revert this and find some better solution around this.

in tracing through how this is used, it is apparently a UUID which can  be used as an auth token to the metrics/healthcheck endpoint.

this makes sense in certain context such as exposing loadbalancer and/or synthetic healthchecks